microsoft-edge/progressive-web-apps-chromium/index.md

```diff
@@ -119,48 +90,45 @@ With you PWAs, you can:
 *   Use CSS layouts, animations, and filters to create advanced designs.
 *   Run near-compiled performance code with WebAssembly.
 
-There isn't much you can't do today with Web technologies, and thanks to Microsoft Edge, PWAs on desktop can take full advantage of this to deliver what users expect desktop apps to do.
+There isn't much you can't do today with web technologies.  Thanks to Microsoft Edge, PWAs on desktop can take full advantage of web technologies to deliver what users expect desktop apps to do.
 
 Check out [Myth Busting PWAs](https://www.davrous.com/2019/10/18/myth-busting-pwas-the-new-edge-edition) for more information about what PWAs can do.
 
 
 <!-- ====================================================================== -->
 ## The Microsoft Store
 
-Because PWAs are just like other apps in the [Microsoft Store](https://www.microsoft.com/store/apps/windows), users can fully engage with them, from discovery, to installation, to execution, without ever opening the browser.
+Because Progressive Web Apps (PWAs) are just like other apps in the [Microsoft Store](https://www.microsoft.com/store/apps/windows), users can fully engage with them—from discovery, to installation, to execution—without<!-- em dashes--> ever opening the browser.
 
-As the most used app on PCs, the Microsoft Store provides a trustworthy and familiar experience for your users to install your app. Additionally, you can view detailed usage statistics and charts that let you know how your apps in the Microsoft Store are doing.
+The Microsoft Store app is the most used app on PCs.  The Microsoft Store provides a trustworthy and familiar experience for your users to install your app.  Additionally, you can view detailed usage statistics and charts that let you know how your apps in the Microsoft Store are doing.
 
-Learn how to [publish your PWA to the Microsoft Store](./how-to/microsoft-store.md).
+Learn how to [publish your PWA to the Microsoft Store](how-to/microsoft-store.md).
 
 
 <!-- ====================================================================== -->
 ## Success stories
 
-Using PWA technologies is a great way to make your app **safe**, **discoverable**, **linkable**, **easy to install** and update, **responsive**, and **network independent**. Many businesses use PWAs and have had success.
+Using Progressive Web App (PWA) technologies is a great way to make your app **safe**, **discoverable**, **linkable**, **easy to install** and update, **responsive**, and **network independent**.  Many businesses have had success with PWAs.  For example:
```

Comment:
We keep using both `Progressive Web App` and `PWA` throughout these articles. And we often put them next to each other: `Progressive Web App (PWA)`. 
Do you think we need to do this? Or could we perhaps use `Progressive Web App` everywhere, and drop `PWA`?
Or, alternatively, use `Progressive Web App (PWA)` on this index page, at the beginning, to help new users understand the term, and then just use `PWA` everywhere else?

Reply:
"We often put them next to each other":
https://docs.microsoft.com/en-us/style-guide/acronyms
"In general, include the acronym in parentheses following the spelled-out term. On subsequent mentions in the same article, page, or screen, you can use the acronym without spelling it out."

"Could we perhaps use Progressive Web App everywhere, and drop PWA?"
Depends: How often is "PWA" used in the industry?  I'm assuming that the industry heavily uses both "PWA" and Progressive Web Application, so I'm keeping both, as-is, which is: in any article that uses "PWA", define it in the 1st occurrence.  I also define it in the first occurrence in each h2 section - that's debatable.

We cannot assume the word "PWA" means anything to the reader; it's safer to use the spelled-out term, at least at top of every article, and possibly at top of each h2 section, as done in this PR.

On the hypertext & Docs principle of "Every page is Page 1", we must assume the reader of any article hasn't seen the "first" article.  Per MWSG, and per standard Docs practice, an acronym must be defined in every article: "On subsequent mentions in the same article, page, or screen, you can use the acronym without spelling it out."

Reply:
Thanks. Having the "Progressive Web Apps (PWAs)" for the first mention of every page makes a lot of sense. I would tend to use PWA only for subsequent mentions though.